### ISSUE TITLE
[Accessibility bug] Buttons with no description

### ISSUE THREAD

**Issue by victoriamdo** [hyperlink-academy/leaflet#206]
When inside a post from a publication, the first smaller buttons for quotes and comments don't have any description, so the screen reader only reads "number button". Lacks context for visually impaired people.

<img width="381" height="288" alt="Image" src="https://github.com/user-attachments/assets/825216dc-b94b-465f-8ed6-bf4c4cc3ce63" />

<img width="380" height="288" alt="Image" src="https://github.com/user-attachments/assets/2af12864-db32-4f03-b7e9-d831f80ccc5d" />

You can reproduce the problem using the screen reader native to your phone, or using a browser that can read out loud webpages (like Edge).
